packages/react/src/components/Dialog/Dialog.test.tsx

```diff
@@ -273,4 +274,42 @@ describe('Dialog', () => {
       wrapper.unmount();
     });
   });
+
+  it('hides siblings when open', () => {
+    const { getByText } = render(
+      <div>
+        <div>sibling</div>
+        <Dialog hidden={false}>content</Dialog>
+      </div>,
+    );
+
+    const bodyChildren = Array.from(document.body.children) as HTMLElement[];
```

Comment:
I don't think this is going to get you the sibling `<div>`, since it'll return an array of top-level elements. Maybe a targeted test that selects the sibling div to check for `aria-hidden` might be better? It could also check that the parent `<div>` doesn't have `aria-hidden`

Reply:
Thank you for noticing that I'm testing nothing 🤦‍♀️

Reply:
Updated the tests so they're actually testing things in a valid way